src/components/Signin.jsx

```diff
@@ -0,0 +1,268 @@
+import React,{useState} from 'react';
+import undraw_login from '../components/assets/images/undraw_Login.svg';
+import styled from 'styled-components';
+
+const Content = styled.div`
+  width: 70%;
+`;
+
+const Button = styled.button`
+  width:${props => props.width};
+  margin-left:${props=>props.marginLeft};
+`;
+
+const AuthLinkWrapper = styled.div`
+  margin:40px;
+  h2 { 
+    width:100%; 
+    text-align:center; 
+    border-bottom: 5px solid gray; 
+    line-height:0.1em; 
+    margin:10px 0 20px;
+  } 
+  h2 span { 
+    background:#fff; 
+    padding:0 10px; 
+  }
+
+`;
+
+const DisplayForLinks = styled.div`
+  display: inline;
+  list-style:none;
+  text-align: center;
+  width: 100%;
+  margin: auto;
+  padding: 0 50px;
+  margin-top: 6px;
+  justify-content: space-around;
+  li{
+    margin:10px;
+  }
+  a{
+      width: 90%;
+  }
+  @media screen and (max-width: 650px) {
+    .col {
+      width: 100%;
+      margin-top: 0;
+    }
+  }
+`;
+
+const AuthLink = styled.a`
+  background-color:${props => props.bgColor} ;
+  color: ${props => props.color};
+  &:hover{
+    background-color:${props => props.hoverBgColor};
+    color:${props => props.hoverColor};
+  }
+  i{
+    color:${props => props.IconColor};
+    padding-right:${props => props.paddingRight};
+    background-color:${props => props.IconBgColor};
```

Comment:
Add flex basis property 25px. align self: middle

Reply:
ok

Reply:
why do you want to add this property, when even thought they are working fine, i checked , the links are totally responsive, and the text doesnt flow out aslo., then why do you want to add ?

Reply:
changed align-self: center not middle, it worked.

Reply:
`Final change` 
 - .btn class has applied by default text-align center to anchor tag. Just overwrite that property by text-align:left.
-  No need to make width 90% remove that.
- apply text-align left to facebook *icon* as by default facebook Icon has by default given center property.
Do these minor changes and your PR is ready.

Reply:
ok i will check

Reply:
1.added text-align: left but there is no difference 
2. removed the width:90% and adjusted the padding to 0 60px;
3. added text align :center for facbook icon , but i have not seen any difference.

Reply:
Read the comment properly. I asked to add text-align left to facebook icon, not center (It is by default center) Also apply padding left:2px that will align it with all the other icons.
Apply padding-left: 20px and padding-right:0 to anchor tag.

Reply:
Read the comment properly. I asked to add text-align left to facebook icon, not center (It is by default center) Also apply padding left:2px that will align it with all the other icons.
Apply padding-left: 20px and padding-right:0 to anchor tag.

Reply:
ok i will check